modules/nw-running-dpdk-rootless-tap.adoc

```diff
@@ -0,0 +1,269 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-running-dpdk-rootless-tap_{context}"]
+= Using the TAP CNI to run a rootless DPDK workload with kernel access
+
+DPDK applications can use `virtio-user` as an exception path to inject certain types of packets for example log messages into the kernel for processing. For more information about this feature see link:https://doc.dpdk.org/guides/howto/virtio_user_as_exception_path.html[Virtio_user as Exception Path].
```

Comment:
with this reference, I think that we definitely don't need the diagram since there is one there.